gremlin-go/driver/connection_test.go

```diff
@@ -1,50 +1,208 @@
+/*
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+*/
+
 package gremlingo
 
 import (
+	"sort"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"golang.org/x/text/language"
 )
 
-const runIntegration = false
+const runIntegration = true
+const testHost string = "localhost"
+const testPort int = 8182
+const personLabel = "Person"
+const testLabel = "Test"
+const nameKey = "name"
+
+func dropGraph(t *testing.T, g *GraphTraversalSource) {
+	// Drop vertices that were added.
+	_, promise, err := g.V().Drop().Iterate()
+	assert.Nil(t, err)
+	assert.NotNil(t, promise)
+	<-promise
+}
+
+func getTestNames() []string {
+	return []string{"Lyndon", "Yang", "Simon", "Rithin", "Alexey"}
+}
```

Comment:
dumb question: any reason not to just define these as a package variable?

ie:

```
package gremlingo

var testNames = []string{"Lyndon", "Yang", "Simon", "Rithin", "Alexey"}
```